resources/import_node.rb

```diff
@@ -14,7 +14,7 @@
   node_assets = {}
   node_category = []
   meta_datas = []
-  meta_data = {}
+  mdata = {}
```

Comment:
this variable would be better scoped inside the each_element('meta-data') lambda